support/hydrationStyles.ts

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
+const path = require("path");
+
+(async function () {
+  const stylePath = path.normalize(`${__dirname}/../src/assets/styles/_hydration.scss`);
+  const components = await fs.promises.readdir(path.normalize(`${__dirname}/../src/components/`));
+  const selectors = components.map((component) => `calcite-${component}:not([calcite-hydrated])`);
```

Comment:
Something like this would be nice...

```ts
const hiddenComponents = new Set<string>(["alert", "input-message", "popover", "tooltip", "tree-item"]);

(async function () {
  const stylePath = path.normalize(`${__dirname}/../src/assets/styles/_hydration.scss`);
  const components = await fs.promises.readdir(path.normalize(`${__dirname}/../src/components/`));
  const selectors = components.map((component) => `calcite-${component}:not([calcite-hydrated])`);
  const selectors = components
    .filter((component) => !hiddenComponents.has(component))
    .map((component) => `calcite-${component}:not([calcite-hydrated])`);
```

Reply:
this would flash unstyled content prior to javascript running